kamaki/clients/compute/rest_api.py

```diff
@@ -401,3 +401,36 @@ def floating_ips_post(self, json_data, ip='', success=200, **kwargs):
     def floating_ips_delete(self, ip='', success=204, **kwargs):
         path = path4url('os-floating-ips', ip or '')
         return self.delete(path, success=success, **kwargs)
+
+    def keypairs_get(self, key_name=None, success=200, **kwargs):
+        """GET endpoint_url/os-keypairs/[<key_name>]
+
+        :param key_name: (str) the name of the key
+
+        :returns: request response
+        """
+        path = path4url('os-keypairs', key_name or '')
+        return self.get(path, success=success, **kwargs)
+
+    def keypairs_post(self, json_data=None, success=200, **kwargs):
```

Comment:
Do we allow empty "POST" requests here?
If yes, what are the semantics?
If not, we should make "json_data" obligatory.